mlflow/store/artifact/dbfs_artifact_repo.py

```diff
@@ -213,7 +212,7 @@ def dbfs_artifact_repo_factory(artifact_uri):
         return DatabricksArtifactRepository(cleaned_artifact_uri)
     elif (
         mlflow.utils.databricks_utils.is_dbfs_fuse_available()
-        and os.environ.get(USE_FUSE_ENV_VAR, "").lower() != "false"
+        and MLFLOW_ENABLE_DBFS_FUSE_ARTIFACT_REPO.get().lower() != "false"
```

Comment:
With https://github.com/mlflow/mlflow/pull/9250/files#r1286953735, we can fix this line like this:

```suggestion
        and MLFLOW_ENABLE_DBFS_FUSE_ARTIFACT_REPO.get()
```

Reply:
Yeah, this is better! Thanks.